source/query-with-natural-language/prompt-natural-language-aggregation.txt

```diff
@@ -0,0 +1,160 @@
+.. _prompt-natural-language-agg:
+
+==============================================
+Prompt your First Natural Language Aggregation
+==============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+You can use MongoDB Compass to generate aggregation queries using natural 
+language. Compass uses AI to generate aggregations based on prompts 
+you provide. Querying with natural language can be a helpful starting 
+point and assist you in learning to write MongoDB queries.
+
+About this Task
+---------------
+
+.. note::
+
+   .. include:: /includes/fact-natural-language-query.rst
+
+- You can also provide natural language prompts on the 
+  :ref:`documents tab <prompt-natural-language-query>`.
+
+- This page uses the :ref:`sample_mflix.movies <sample-mflix>` 
+  collection from the Atlas sample dataset.
+
+Before you Begin
+----------------
+
+You must :ref:`enable natural language querying in Compass<enable-natural-language-querying>`.
+
+Steps
```

Comment:
is it worth calling out somewhere that this experience can happen on both the text editor and stages view?